cdap-docs/cdap-apps/source/hydrator/custom.rst

```diff
@@ -450,18 +453,30 @@ Plugin Deployment
    :start-after: .. _plugins-deployment-artifact:
    :end-before:  .. _plugins-use-case:
 
-Configuration JSON for a plugin (for Webapp UI alone)
-======================================================
-Every property of a plugin is represented, by default, as input fields in the UI. This can be however changed to specific widgets that closely match the type of the property in a plugin. This can be specified as JSON for the UI to consume.
+.. _cdap-apps-custom-plugin-configuration-json:
+
+Plugin Configuration JSON
+-------------------------
+When a plugin is displayed in the CDAP UI, the plugin and its properties are represented
```

Comment:
May not be plugin & its properties. Its just properties that are represented as widgets.